3648/3/db/seeds.rb

```diff
@@ -0,0 +1,15 @@
+require 'csv'
+
+NAME = 0
+LOCATION = 1
+DESCRIPTION = 2
+RATING = 3
+
+CSV.foreach(File.realpath('db/data/places.csv')) do |row|
+  Place.find_or_create_by(
```

Comment:
when I run seeds changing only description, I get duplicates (multiple places with the same name).
I want to identify place by name only.
Change seeds so that it find_or_create by name only
and then update other fields

Reply:
can you see now